java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -212,4 +212,46 @@ public interface ServerManagementClusterCommands {
      * }</pre>
      */
     CompletableFuture<String> configSet(Map<String, String> parameters, Route route);
+
+    /**
+     * Returns the server time.<br>
+     * The command will be routed to a random node.
+     *
+     * @see <a href="https://redis.io/commands/time/">redis.io</a> for details.
+     * @return The current server time as a <code>String</code> array with two elements: A Unix
+     *     timestamp and the amount of microseconds already elapsed in the current second. The
+     *     returned array is in a [Unix timestamp, Microseconds already elapsed] format.
```

Comment:
maybe wrap with `code` this block `[Unix timestamp, Microseconds already elapsed]`

Reply:
ok